python/pyspark/mllib/classification.py

```diff
@@ -600,12 +604,15 @@ class StreamingLogisticRegressionWithSGD(StreamingLinearAlgorithm):
     :param miniBatchFraction: Fraction of data on which SGD is run for each
                               iteration.
     :param regParam: L2 Regularization parameter.
+    :param convergenceTol: A condition which decides iteration termination.
```

Comment:
Yes, I think I can do these in the follow-up PR because I found that there are much room to improve for ```StreamingLogisticRegressionWithSGD``` of PySpark.